src/test/java/oss/fosslight/controller/SelfCheckControllerTest.java

```diff
@@ -0,0 +1,102 @@
+package oss.fosslight.controller;
```

Comment:
SelfCheckControllerTest and SelfCheckControllerIntegrationTest is same. Please check it, @Lee-JaeHyuk